scripts/zones/Xarcabard/mobs/Barbaric_Weapon.lua

```diff
@@ -0,0 +1,15 @@
+-----------------------------------
+-- Area: Xarcabard
+--   NM: Barbaric Weapon
+-- TODO: "Gains a Dread Spikes effect whenever Whirl of Rage is used."
+-----------------------------------
+require("scripts/globals/status")
+-----------------------------------
+
+function onMobInitialize(mob)
+    mob:setMod(dsp.mod.DOUBLE_ATTACK, 10)
```

Comment:
He's war/war in his pool entry and should already have the base double attack from that. if he needs additional double attack on top of that you'll need to addMod